app/assets/stylesheets/components/user_settings.scss

```diff
@@ -0,0 +1,114 @@
+#user_settings{
+  line-height: 1.88;
+  letter-spacing: .2px;
+  color: $text-primary;
+  padding-bottom: 110px;
+  .settings-card {
+    background: $white;
+    max-width: 780px;
+    padding: 22px 32px;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.3)
+  }
+  .settings-section {
+    max-width: 780px;
+    margin: 0 auto 70px;
+  }
+  .profile-pic {
+    margin-right: 32px;
+    display: flex;
+    align-items: center;
```

Comment:
Nit. You can remove this definition and use the `align-items-center` class that ships with bootstrap 4.